src/ios/CDVWKInAppBrowser.m

```diff
@@ -917,6 +924,17 @@ - (void)createViews
         configuration.mediaPlaybackRequiresUserAction = _browserOptions.mediaplaybackrequiresuseraction;
     }
 
+    // preferred content mode for iPad's. Since iOS 13 iPad's default to desktop mode.
```

Comment:
OFC they do - why would a tablet not be a desktop. I mean it's a an _Apple_ tablet, so that counts the same as a desktop from a "lesser" manufactorer.

/s for future comment reviewing code-archaeologists. I hate Apple.

Reply:
"I hate Apple" The nicest words ever given to Apple from a dev.